src/config/texts.js

```diff
@@ -908,7 +909,23 @@ export const texts = {
     currentStatus: 'Aktueller Status',
     datetime: 'Datum und Uhrzeit der Meldung',
     description: 'Beschreibung',
-    location: 'Ort'
+    location: 'Ort',
+    reportScreen: {
```

Comment:
`report` should be enough, we do not have this `...Screen` pattern here, or am i wrong?

Reply:
That's right. I'll change that.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/966/commits/2b603bd8f6f8b43bf449efd90fce208da3b2c23c